example.py

```diff
@@ -31,74 +26,60 @@
 type_column = 'colType'
 model_weights_path = 'torontobooks_unigrams.bin'
 
-open_d3m_file = 'data/open_d3m_data.csv'
-closed_d3m_file = 'data/closed_d3m_data.csv'
+open_d3m_file = '~/data/open_d3m_unembed_data.csv'
+closed_d3m_file = '~/data/closed_d3m_unembed_data.csv'
```

Comment:
can we add the data to this repo? maybe a compressed version.

Reply:
can we also rename `open` to `public` and `closed` to `private`

Reply:
@bjschoenfeld is it okay if the metadata for the private database (`closed_d3m_unembed_data.csv` above) is hosted on our repo? It's not _the_ data, but it includes all the private dataset names and descriptions.

Reply:
@bjschoenfeld I uploaded a compressed version of the public metadata `.csv` file. Let me know if I have the go-ahead to do the same for the private metadata `.csv` file too.

Reply:
maybe to be safe, let's not publish the private one.